books/spiders/books_list.py

```diff
@@ -0,0 +1,59 @@
+import scrapy
+from scrapy import Selector
+from scrapy.http import Response
+from enum import Enum
+
+from books.items import BooksItem
+
+
+class RatingEnum(Enum):
+    Zero = 0
+    One = 1
+    Two = 2
+    Three = 3
+    Four = 4
+    Five = 5
+
+
+def get_rating_value_from_class_names(class_names: str) -> int | None:
+    class_names_list = class_names.split()
+    rating_value_class = class_names_list[-1]
+    return RatingEnum[rating_value_class].value
```

Comment:
The `get_rating_value_from_class_names` function assumes that the last class name will always match a member of `RatingEnum`. Consider adding error handling to manage cases where the class name does not match any enum member, which could raise a `KeyError`.